silverback/application.py

```diff
@@ -64,23 +66,25 @@ def __init__(self, settings: Settings | None = None):
         self.tasks: defaultdict[TaskType, list[TaskData]] = defaultdict(list)
         self.poll_settings: dict[str, dict] = {}
 
-        atexit.register(self.network.__exit__, None, None, None)
+        atexit.register(network.__exit__, None, None, None)
 
         self.signer = settings.get_signer()
         self.new_block_timeout = settings.NEW_BLOCK_TIMEOUT
         self.start_block = settings.START_BLOCK
 
-        network_str = f'\n  NETWORK="{provider.network.ecosystem.name}:{provider.network.name}"'
+        self.network_choice = f"{provider.network.ecosystem.name}:{provider.network.name}"
         signer_str = f"\n  SIGNER={repr(self.signer)}"
         start_block_str = f"\n  START_BLOCK={self.start_block}" if self.start_block else ""
         new_block_timeout_str = (
             f"\n  NEW_BLOCK_TIMEOUT={self.new_block_timeout}" if self.new_block_timeout else ""
         )
         logger.info(
-            f"Loaded Silverback App:{network_str}"
+            f'Loaded Silverback App:\n  NETWORK="{self.network_choice}"'
             f"{signer_str}{start_block_str}{new_block_timeout_str}"
         )
 
+        self.state = None  # Runner manages this
```

Comment:
issue(non-blocking): I don't really like this.  It's runner-only state that's stored on the application for some reason, using the same terminology as worker state.  Seems a bit confusing.

Reply:
suggestion: leave it to the runner?  Unless there's some reason other parts need access to this.